src/sounds.cpp

```diff
@@ -422,7 +422,7 @@ void sounds::process_sound_markers( player *p )
         if( p->is_npc() ) {
             if( !sound.ambient ) {
                 npc *guy = dynamic_cast<npc *>( p );
-                guy->handle_sound( static_cast<int>( sound.category ), description,
+                guy->handle_sound( sound.category, description,
                                    heard_volume, pos );
```

Comment:
```suggestion
```

Reply:
Huh, not sure how I missed this, good eye. Will have to pay more attention to recombining short lines in the future.